auth0_flutter_platform_interface/lib/src/user_profile.dart

```diff
@@ -163,4 +163,28 @@ class UserProfile {
                 result['custom_claims'] as Map<dynamic, dynamic>)
             : null,
       );
+
+  Map<String, dynamic> toMap() => {
+        'sub': sub,
+        'name': name,
+        'given_name': givenName,
+        'family_name': familyName,
+        'middle_name': middleName,
+        'nickname': nickname,
+        'preferred_username': preferredUsername,
+        'profile': profileUrl?.toString(),
+        'picture': pictureUrl?.toString(),
+        'website': websiteUrl?.toString(),
+        'email': email,
+        'email_verified': isEmailVerified,
+        'gender': gender,
+        'birthdate': birthdate,
+        'zoneinfo': zoneinfo,
+        'locale': locale,
+        'phone_number': phoneNumber,
+        'phone_number_verified': isPhoneNumberVerified,
+        'address': address,
+        'updated_at': updatedAt?.toUtc(),
```

Comment:
My bad, I didn't communicate this clearly. I meant the following:
`updatedAt?.toUtc().toIso8601String()`. That is, the `DateTime` is converted to UTC before being stringified.